internal/gke/deploy.go

```diff
@@ -956,11 +954,90 @@ func createServiceAccountsIAMPolicyRole(ctx context.Context, config CloudConfig)
 	})
 }
 
+// getConfigCluster computes the region and the cluster that manages all the
+// applications within a Service Weaver project, and the cluster .
```

Comment:
fix the comment.

Also, can we have it return name, region, error, in that order?

Reply:
Done